distribution/ltp-upstream/lite/runtest.sh

```diff
@@ -58,7 +58,7 @@ function ltp_test_build()
 		pushd ltp; git pull  > /dev/null 2>&1; popd
 	else
 		if [ $GIT_USER_ADDR = 0 ]; then
-			git clone https://github.com/linux-test-project/ltp ltp  --depth=1\
+			git clone https://github.com/linux-test-project/ltp ltp \
```

Comment:
Why remove the `--depth=1`?

Reply:
@emanchado I noticed that sometimes it would fail when using the `--depth=1` option, I saw the following in the logs` fatal: reference is not a tree: c9707b465a08397569920b676345474f44327200` I don't know why but removing it fixed it.

Reply:
Ah, spicy, "one of those"...